src/main/java/org/candlepin/subscriptions/metering/service/prometheus/MetricProperties.java

```diff
@@ -44,6 +49,11 @@
      */
     private int rangeInMinutes = 60;
 
+    /**
+     * Latency offset: how many minutes in the past you would like the metering job latency (window) to occur.
+     */
+    private int offsetInMinutes = 60;
```

Comment:
I'd like to make sure this is consistent with the existing `prometheusLatencyDuration` property.

- [ ] Instead of defining this here, can you please move to ApplicationProperties for consistency?
- [ ] Can you please define as a `Duration`?
- [ ] Can you please rename to reflect that this offset applies to the range of time for the hourly tally (e.g. `hourlyTallyOffset`?)
- [ ] Can you please leave a comment about why this offset is needed?
- [ ] Can you please define an environment variable for this in `src/main/resources/application.yaml` ?